Solutions/ESI - Exchange Security Config and Audit Insights/Analytic Rules/ServerOrientedWithUserOrientedAdministration.yaml

```diff
@@ -0,0 +1,88 @@
+id: 7bce901b-9bc8-4948-8dfc-8f68878092d5
+name: Server Oriented Cmdlet And User Oriented Cmdlet used
+description: |
+  'Detect if a server oriented Cmdlet and a user oriented cmdlet that are monitored are launched by a same user in a same server in a 10 minutes timeframe'
+requiredDataConnectors:
+  - connectorId: ESI-ExchangeAdminAuditLogEvents
+    dataTypes:
+      - Event
+severity: High
+queryFrequency: 5h
+queryPeriod: 5h
+triggerOperator: gt
+triggerThreshold: 0
+status: Available
+tactics:
+  - Exfiltration
+  - Persistence
+  - Collection
+relevantTechniques:
+  - T1020
+  - T1098
+  - T1114
+query: |
+  let timeframe = 1d;
+  let spanoftime = 10m;
+  let threshold = 0;
+  ExchangeAdminAuditLogs 
+    | where TimeGenerated > ago(2 * timeframe)
+    | where isempty(UserOriented)
+    | project serverExecutedTime = TimeGenerated,
+      ServerCmdlet = CmdletName,
+      ServerCmdletParams = CmdletParameters,
+      Computer,
+      Caller,
+      ServerCmdletTargetObject = TargetObject
+    | join kind= inner (
+        ExchangeAdminAuditLogs
+        | where TimeGenerated > ago(timeframe)
+        | where UserOriented == 'Yes'
+        | lookup kind=leftouter _GetWatchlist('ExchangeVIP') on $left.TargetObject == $right.canonicalName
+        | project userExecutedTime = TimeGenerated,
+          UserCmdlet = CmdletName,
+          UserCmdletParams = CmdletParameters,
+          Computer,
+          Caller,
+          UserCmdletTargetObject = TargetObject,
+          userPrincipalName,
+          objectGUID,
+          sAMAccountName,
+          IsVIP)
+      on Computer, Caller
+    | where userExecutedTime - serverExecutedTime < spanoftime
+    | extend TimeDelta = userExecutedTime - serverExecutedTime
+    | extend TimeDeltaInverse = serverExecutedTime - userExecutedTime
+    | where tolong(TimeDelta) >= threshold or tolong(TimeDeltaInverse) >= threshold
+entityMappings:
+- entityType: Mailbox
```

Comment:
Same comment as before, duplicate values will not add extra value. Please add unique values in each entity, unless required otherwise.

Reply:
What is the BP when the selected column can contains either one identifier or another one ? If my understand is well, the goal of this mapping is to be able to connect multiple alerts between them using entities. If I put only 1 mapping, we will loose this interconnexion in logs based on the other mapping.

So what you preconise here ?